jdbc/pom.xml

```diff
@@ -33,6 +33,144 @@
   <version>0.9.0-SNAPSHOT</version>
   <name>Zeppelin: JDBC interpreter</name>
 
+  <properties>
+    <!--library versions-->
+    <interpreter.name>jdbc</interpreter.name>
+    <postgresql.version>9.4-1201-jdbc41</postgresql.version>
+    <hadoop.common.version>2.7.2</hadoop.common.version>
+    <h2.version>1.4.190</h2.version>
+    <commons.dbcp2.version>2.0.1</commons.dbcp2.version>
```

Comment:
why do we use version that is released in 2014th? Maybe it makes sense to upgrade to some later version?

Reply:
Yes, agree.